libs/perun/dialogs/src/lib/password-reset-dialog/password-reset-dialog.component.html

```diff
@@ -0,0 +1,60 @@
+<ng-template #spinner>
+  <perun-web-apps-loading-dialog></perun-web-apps-loading-dialog>
+</ng-template>
+<div class="user-theme position-relative">
+  <div class="title-with-icon">
+    <h1 mat-dialog-title>
+      {{'SHARED_LIB.PERUN.COMPONENTS.RESET_PASSWORD_DIALOG.TITLE' | translate}}
+    </h1>
+  </div>
+  <div class="dialog-container" mat-dialog-content>
+    <p *ngIf="getLogin()" class="subtitle">
```

Comment:
Please don't use functions to preform data calculations, because of how change detection works, this will be recalculated thousand of times. It you need to transform data somehow use a pipe, otherwise just use a variable. The information is passes to the dialog and can be directly used here.